README.md

```diff
@@ -50,22 +50,22 @@ This library is designed to work with Node 8/ES5/CommonJS.
 1. Use the library to write Ion data:
     ```javascript
     import * as ion from 'ion-js';
+    import {IonTypes} from 'ion-js';
     import {cryptoIonHasherProvider, makeHashWriter} from 'ion-hash-js';
-    
+
     let hashWriter = makeHashWriter(
             ion.makeTextWriter(),
             cryptoIonHasherProvider('md5'));
-    hashWriter.writeList();
+    hashWriter.stepIn(IonTypes.LIST);
     hashWriter.writeInt(1);
     hashWriter.writeInt(2);
     hashWriter.writeInt(3);
-    hashWriter.endContainer();
+    hashWriter.stepOut();
     let digest = hashWriter.digest();
-    
+
     process.stdout.write('digest:  ');
-    digest.forEach(b => {
-        process.stdout.write(('0' + ((b as number) & 0xFF)
-                .toString(16)).slice(-2) + ' ');
+    digest.forEach((b: number) => {
+        process.stdout.write(('0' + (b & 0xFF).toString(16)).slice(-2) + ' ');
```

Comment:
Two minor thoughts:
* This snippet appears twice in the README. Would it be worth providing it as a `toPrettyString()` method?
* When iterating over `digest.forEach((b: number) => {})`, can `b` ever be greater than `0xFF`?

Reply:
* it would be tangential to the library, as well as open the door to providing n pretty-print variations; opting to not provide such a utility method at this time
* `b` represents a single byte and should never be greater than `0xFF`